PS-8889 Add quick start topic (8.0) (#210)
percona/psmysql-docs: docs/quickstart-docker.md

The following command retrieves the `8.0.34` version.

```{.bash data-prompt="$"}
$ docker pull percona/percona-server:8.0.34
```

List the downloaded Docker image with the following command:

```{.bash data-prompt="$"}
$ docker images
```

??? example "Expected output"

    ```{.text .no-copy}
    REPOSITORY               TAG       IMAGE ID       CREATED        SIZE
    percona/percona-server   8.0.34    b2588da614b1   2 months ago   1.33GB
    ```

## Create a volume

Create a volume for persistent storage for the data.

```{.bash data-prompt="$"}
$ docker volume create myvol
```

## Run the container

The `docker run` command starts a new container with that image.

```{.bash data-prompt="$"}
$ docker run -d --name=psmysql -e MYSQL_ROOT_PASSWORD=secret --volume=myvol:/var/lib/mysql percona/percona-server:8.0.34
```

??? example "Expected output"

    ```{.text .no-copy}
    9d0acf24fcf66ad01fcc19067206eada4e2303dad039bb020daa2bd0b2e55d59
    ```

We have added the following options:

* `--name` - provides a selected name to the container. If you do not use this option, docker provides a random name.
* `-d` - the container runs in the background in detached mode.

> * `-e` - to start a Percona Server for MySQL container use the `MYSQL_ROOT_PASSWORD` environment variable. The instance refuses to initialize without these variables. Choose a more secure password, if needed.

Reviewer: ```suggestion
* `-e` - to start a Percona Server for MySQL container, use the `MYSQL_ROOT_PASSWORD` environment variable. The instance refuses to initialize without this variable. Choose a more secure password if needed.
```

* `--volume` - for persistent storage of the database data.
* `8.0.34` - a tag that specifies the release version.

You must have either the `MYSQL_ROOT_PASSWORD`, the `MYSQL_ALLOW_EMPTY_PASSWORD` or the `MYSQL_RANDOM_ROOT_PASSWORD` environment variables or the instance refuses to initialize.

The `docker ps -a` command displays a list of all containers with information about the containers. 

```{.bash data-prompt="$"}
$ docker ps -a
```

??? example "Expected output"

    ```{.text .no-copy}
    CONTAINER ID   IMAGE                           COMMAND                  CREATED              STATUS              PORTS                 NAMES
    9d0acf24fcf6   percona/percona-server:8.0.34   "/docker-entrypoint.…"   About a minute ago   Up About a minute   3306/tcp, 33060/tcp   psmysql
    ```

You can inspect the volume.

```{.bash data-prompt="$"}

> docker inspect psmysql | grep -i volume

Reviewer: ```suggestion
$ docker inspect psmysql | grep -i volume
```

```

??? example "Expected output"

    ```{.text .no-copy}
    "VolumeDriver": "",
                "VolumesFrom": null,
                    "Type": "volume",
                    "Source": "/var/lib/docker/volumes/myvol/_data",
                    "Type": "volume",
                    "Source": "/var/lib/docker/volumes/10b10b0f6041c401ac64a9d114e2ddb54232763ecc10821a495bebac097ce7cd/_data",
                "Volumes": {
    ```

## Connect to the container

Connect with the `docker exec` command to run commands in a running container.

```{.bash data-prompt="$"}
$ docker exec -it psmysql /bin/bash
```

We have the following options:

> * `it` - Interactive and be a pseudo-terminal

Reviewer: ```suggestion
* `it` - interactive and be a pseudo-terminal
```

* `psmysql` - the name of the running container
* `\bin\bash` - the shell used for running commands

## Connect to the instance

Connect to the database server instance with the following command:

```{.bash data-prompt="$"}
$ mysql -uroot -p
```

> 

Reviewer: We need to add explanations about the password. that we use secret

Author: Line 155
* `-p` - the password to use when connecting to the server. Enter the password when prompted by the server.
